python/pyspark/sql/dataframe.py

```diff
@@ -1722,8 +1806,17 @@ def dtypes(self) -> List[Tuple[str, str]]:
 
         Examples
         --------
+        >>> df = spark.createDataFrame([(14, "Tom"),
+        ...     (23, "Alice")], ["age", "name"])
```

Comment:
I think you can inline here

Reply:
what do you mean by inline?

Reply:
nvm, I remember in the other example you had [14, "tom"] in the same line as the others